cmd/storagenode/gracefulexit.go

```diff
@@ -205,13 +205,18 @@ func cmdGracefulExitStatus(cmd *cobra.Command, args []string) (err error) {
 }
 
 func displayExitProgress(w io.Writer, progresses []*pb.ExitProgress) {
-	fmt.Fprintln(w, "\nDomain Name\tNode ID\tPercent Complete\tSuccessful")
+	fmt.Fprintln(w, "\nDomain Name\tNode ID\tPercent Complete\tSuccessful\tCompletion Receipt")
```

Comment:
nit - I wonder if this should just be `Receipt`.  The word `Completion` might imply success...even though there is a `Successful` column that could say otherwise. Thoughts?